tools/abel/abel_tests/guid_to_entity_map_test.py

```diff
@@ -19,11 +19,11 @@
 from model.entity import VirtualEntity
 from model.guid_to_entity_map import GuidToEntityMap
 from model.site import Site
-from tests.test_constants import TEST_REPORTING_ENTITY_DICT
-from tests.test_constants import TEST_REPORTING_GUID
-from tests.test_constants import TEST_SITE_DICT
-from tests.test_constants import TEST_SITE_DICT_NO_GUID
-from tests.test_constants import TEST_VIRTUAL_ENTITY_DICT
+from abel_tests.test_constants import TEST_REPORTING_ENTITY_DICT
```

Comment:
any reason why this was changed? 
we have the `tests` folder used in all the components.

Reply:
Done, renamed back to `tests`